src/Misc/externals.sh

```diff
@@ -4,7 +4,9 @@ EXTERNALTOOLSNAME=(
     azcopy
     nuget
     pdbstr
-    portablegit
+    portablenixgit
```

Comment:
since I'm still creating this one - will the code skip pulling in the tool if it doesn't exist yet?